gym/spec/platform_detection_spec.rb

```diff
@@ -24,4 +24,22 @@
     expect(Gym.building_for_mac?).to eq(true)
     expect(Gym.building_multiplatform_for_mac?).to eq(true)
   end
+
+  it "detects the correct platform for a visionOS project", requires_xcodebuild: true do
+    options = { project: "./gym/examples/visionos/VisionExample.xcodeproj", sdk: "xros", skip_package_dependencies_resolution: true }
+    Gym.config = FastlaneCore::Configuration.create(Gym::Options.available_options, options)
```

Comment:
maybe we could write

```
  it "...", requires_xcodebuild: true, if: FastlaneCore::Helper.xcode_at_least?('15.0') do
```

and get rid of the if/else.
?

Reply:
Sure, thanks!